src/suites/ported_driver_ca_checks.qls

```diff
@@ -5,7 +5,10 @@
   from: microsoft/windows-drivers

```

Comment:
What is the purpose of putting all ported queries in a suite?

Reply:
In addition to being helpful as a quick way to check what's been ported, it'll be useful when we do wider CA vs CodeQL analyses to be able to run all our ported queries together without dealing with the noise of running unrelated queries.